### DISCUSSION TITLE
PCAPs only in one direction?

### DISCUSSION THREAD

**Discussion by Sazerak** [Security-Onion-Solutions/securityonion#7030] [Unsupported Versions] [answered]
After using SO2 for over a year now, I've finally had a reason to really dig into my PCAPs to try and figure something out, only to realize that my PCAPs seem to only capture incoming traffic, not outgoing. My Hunts seem to able to display 2-way traffic, so I can't imagine the issue is with my network setup. Does anyone have any ideas what I should be looking for to fix this?

**Comment by dougburks** [comment, marked as answer]
I'd start with running `tcpdump` to look at the traffic coming into the sniffing interface and see what it looks like there.

**Reply by Sazerak** [reply]
Just checked and looks like tcpdump is only capturing in one direction too

**Reply by dougburks** [reply]
That seems like an issue with your tap or span port.

**Reply by Sazerak** [reply]
Any suggestions on troubleshooting steps from here? I'm not seeing any issues that jump out at me.

**Reply by dougburks** [reply]
You could try contacting the manufacturer of your tap or switch.

**Reply by Sazerak** [reply]
Are we sure though that the issue is at the span port? I'm seeing two way traffic under Hunt. Shouldn't that mean that the port is working?

**Reply by Sazerak** [reply]
Ah, looks like you were right. I still don't understand how Hunt was able to see 2-way traffic, but I was able to find this link that helped. The TL;DR is that when using an ESXi setup, set the monitoring interface to VLAN ID 4095.

https://community.ui.com/questions/Problem-with-Port-Mirroring-please-help/ee8d4a94-1916-4b8a-a364-f69aa2f9c9fe

**Reply by Sazerak** [reply]
Just out of curiosity, has anyone ever brought this up before? I'm guessing many people must run SO2 in a virtual environment so I can't imagine I'm the first to run into this.
